glean-core/Cargo.toml

```diff
@@ -41,3 +41,7 @@ env_logger = { version = "0.7.1", default-features = false, features = ["termcol
 tempfile = "3.1.0"
 iso8601 = "0.4"
 ctor = "0.1.12"
+
+[features]
+# Enable the "safe-mode" Rust storage backend instead of the default LMDB one.
+rkv-safe-mode = []
```

Comment:
Does this mean the new glean-core version, once we cut a release, will use the new storage system? What happens with the legacy files? Do we need migration?

Reply:
No, it explicitly means we're staying on LMDB.

This feature allows us to toggle it on if we want to (which means we can more easily test this)